src/Content/MailArchive/MailArchiveAttachmentCollection.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Frosh\MailArchive\Content\MailArchive;
+
+use Shopware\Core\Framework\DataAbstractionLayer\EntityCollection;
+
+/**
+ * @extends EntityCollection<MailArchiveAttachmentEntity>
+ */
+class MailArchiveAttachmentCollection extends EntityCollection
```

Comment:
I would prefer not to have a Collection class when it does not have any method. We can just type `EntityCollection<MailArchiveAttachmentEntity>` and PHPStan is happy

Reply:
I like dedicated Collections more than magic typehints, like `EntityRepository<EntityCollection<MailArchiveAttachmentEntity>>`

Reply:
One useless file more in opcache and to autoload. I am fine when we want to keep it